Makefile.am

```diff
@@ -12,6 +12,9 @@ EXTRA_DIST		= CONTRIBUTING.md	\
 			  INSTALL.md		\
 			  README.markdown	\
 			  autogen.sh		\
+			  tap-driver.sh \
+			  tap-test \
+			  m4/glibtests.m4 \
```

Comment:
> maybe add a comment in the makefile where these files came from so we can check for updates occasionally

Can do.

> Looking at glib-tap.mk, I'm not sure whether it should be in EXTRA_DIST or a nodist variable. Our only other current Makefile include, Makefile.common, isn't listed anywhere. (As an aside, I think we should rename Makefile.common as common.mk.) I think I'd like to keep make includes out of the top level, we could maybe put them in include/*.mk, in a new mk/*.mk subdirectory for the purpose, or in a new devel/*.mk subdirectory to put random stuff that's only important to developers (vs sysadmins compiling the code).

I like mk/*.mk, personally.  We don't need to list it in EXTRA_DIST, and Makefile.common is also okay because anything included in a Makefile.am automatically gets included in the source distribution (https://www.gnu.org/software/automake/manual/html_node/Basics-of-Distribution.html).

> Do tap-driver.sh and tap-test need to be in the toplevel directory, or can they go in tests?

They're both referenced in glib-tap.mk so if we are okay with modifying that file, they can go wherever.